_data/concepts.yml

```diff
@@ -89,4 +89,6 @@ toc:
     - docs/concepts/policy/security-context.md
     - docs/concepts/policy/pod-security-policy.md
 
-
+- title: Extensions
```

Comment:
I'm creating a subsection in Concepts called *Extending the Kubernetes API*, so we should group this in that as well.

```
- title: Extending the Kubernetes API
  section:
  - docs/concepts/api-extension/apiserver-aggregation.md  
```